libs/ledger-live-common/src/hw/customLockScreenLoad.ts

```diff
@@ -213,6 +212,14 @@ export default function loadImage({ deviceId, request }: Input): Observable<Load
           sub.unsubscribe();
         };
       }),
+  ).pipe(
+    timeout(5000),
+    catchError(err => {
+      if (err.name === "TimeoutError") {
+        return throwError(() => new DisconnectedDevice());
```

Comment:
The `throwError` in operator is normally not necessary, look at [here](https://rxjs.dev/api/index/function/throwError#unnecessary-usage).

Reply:
Hey @jiyuzhuang! I wrote that to show disconnection error after 5 seconds of timeout